code/modules/mining/machine_redemption.dm

```diff
@@ -10,7 +10,7 @@
 	anchored = TRUE
 	input_dir = NORTH
 	output_dir = SOUTH
-	req_access = list(ACCESS_MINERAL_STOREROOM)
+	req_access = list(ACCESS_MINERAL_STOREROOM, ACCESS_MEDICAL, ACCESS_ARMORY, ACCESS_CONSTRUCTION)
```

Comment:
Wouldn't that make it require ALL listed access types?

Reply:
I've been asking in Discord and the one person who's answered didn't know. The Release option's code is as follows:

>if(check_access(inserted_id) || allowed(usr)) //Check the ID inside, otherwise check the user
                if(params["id"] == "all")
                    materials.retrieve_all(get_step(src, output_dir))
                else

Not sure if this means it's looking for every access or only a singular one

Reply:
Just test it ingame. If it wouldn't work as you intend it to, there's `req_one_access` var.